voltage/internals/http.py

```diff
@@ -536,6 +536,19 @@ async def delete_message(self, channel_id: str, message_id: str):
         """
         return await self.request("DELETE", f"channels/{channel_id}/messages/{message_id}")
 
+    async def bulk_delete_messages(self, channel_id: str, message_ids: List[str]):
+        """
+        Deletes messages in bulk
+
+        Parameters
+        ----------
+        channel_id: :class:`str`
+            The id of the channel.
+        message_ids: :class:`List[str]`
+            The id of the message.
+        """
+        return await self.request("DELETE", f"channels/{channel_id}/messages/{message_ids}")
```

Comment:
According to the docs you pass the message IDs as a JSON object in the body of the request

Reply:
Utilizing json.dumps() for this now

Reply:
that won't work, you have to pass it into the body of the request